pages/services/confluent-kafka/2.6.0-5.1.2/configuration/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: layout.pug
+navigationTitle: Configuring 
+excerpt: Configuring Confluent Kafka
+title: Configuring 
+menuWeight: 20
+model: /services/confluent-kafka/data.yml
+render: mustache
+---
+
+#include /services/include/configuration-install-with-options.tmpl
+#include /services/include/configuration-service-settings.tmpl
+#include /services/include/configuration-regions.tmpl
+
+## Configuring the ZooKeeper Connection.
+
+{{ model.techName }} requires a running ZooKeeper ensemble to perform its own internal accounting. By default, the DC/OS {{ model.techName }} Service uses the ZooKeeper ensemble made available on the Mesos masters of a DC/OS cluster at `master.mesos:2181/dcos-service-<servicename>`. At install time, you can configure an alternate ZooKeeper for {{ model.techName }} to use. This enables you to increase {{ model.techName }}'s capacity and removes the DC/OS System ZooKeeper ensemble's involvement in running it.
+
+<p class="message--note"><strong>NOTE: </strong>If you are using the <a href="/services/confluent-zookeeper/">DC/OS Apache ZooKeeper service</a>, use the DNS addresses provided by the <tt>dcos confluent-zookeeper endpoints clientport</tt> command as the value of <tt>kafka_zookeeper_uri</tt>.</p>
+To configure an alternate Zookeeper instance:
+
+1. Create a file named `options.json` with the following contents. Here is an example `options.json` which points to a `confluent-zookeeper` instance named `confluent-zookeeper`:
```

Comment:
This code block contains commands for "kafka-zookeeper", not "confluent-zookeeper". Is this correct?

Reply:
yes, `kafka_zookeeper_uri` is a configuration option of confluent-kafka which should be filled with the result of `dcos confluent-zookeeper endpoints clientport`

so this is correct

Reply:
Thank you.